conda-env.yml

```diff
@@ -6,7 +6,6 @@ dependencies:
   - python=3.10
   - click>=8.1.3,<8.2
   - numpy>=1.22,<1.23
-  - datalad>=0.15.4,<0.18
```

Comment:
This should not change. We need to create a new conda-env for apple macs, but not change this one.